h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -39,13 +42,26 @@ private GLMModelOutputV3 fillMultinomial(GLMOutput impl) {
       String [] ns = ArrayUtils.append(new String[]{"Intercept"},Arrays.copyOf(names,names.length-1));
 
       coefficients_table = new TwoDimTableV3();
+      if (impl._responseColumn >= 0) // _responseColumn -1 for regression and binomial.
```

Comment:
can you use _nClass > 2 instead? do we really need _responseColumn>

Reply:
Yes, changed it.